### PR TITLE
qa: Replaced centos_7.3.yaml with centos_latest.yaml

### PR DESCRIPTION
**Pull request by yuriw**
Changed 7.3 to 7.4 for rbd (hardcoded @dillaman FYI)
And upgrade/jewel-x/point-to-point-x/distros/ubuntu_14.04.yaml

Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

File: qa/suites/upgrade/jewel-x/point-to-point-x/distros/ubuntu_14.04.yaml [hunk 1]
```diff
@@ -1 +0,0 @@
-../../../../../distros/all/ubuntu_14.04.yaml
```
Comment (ktdreyer) [inline qa/suites/upgrade/jewel-x/point-to-point-x/distros/ubuntu_14.04.yaml]: Might want to call out in the commit message that Ubuntu is changing here as well

Comment (yuriw) [reply qa/suites/upgrade/jewel-x/point-to-point-x/distros/ubuntu_14.04.yaml]: oh forgot that :(

Comment (smithfarm) [reply qa/suites/upgrade/jewel-x/point-to-point-x/distros/ubuntu_14.04.yaml]: Jewel is only tested on 14.04, so the impact of this change is unknown.

**Comment by smithfarm** [comment]
This needs an `upgrade/jewel-x/point-to-point` run.

File: qa/suites/upgrade/jewel-x/point-to-point-x/distros/ubuntu_14.04.yaml (hunk 1 above)
Comment (yuriw) [reply qa/suites/upgrade/jewel-x/point-to-point-x/distros/ubuntu_14.04.yaml]: @smithfarm shall we test or undo?

Comment (smithfarm) [reply qa/suites/upgrade/jewel-x/point-to-point-x/distros/ubuntu_14.04.yaml]: Ubuntu 14.04 is not going away, so I don't see why we need to migrate jewel away from it.

**Comment by liewegas** [comment]
needs rebase...

**Comment by liewegas** [comment]
This is a stale pull request and the head branch is in ceph.git.  If this is still a useful change, please reopen a new pull request from your own cloned repo (not the main ceph.git).  You can find the original branch archived in ceph-old.git.  Thanks!

**Pull request closed without merging**